### PR TITLE
[frontend] fix filters adding for TimeLine views, Alert Live, and label icon (#6348)

### LINKED ISSUE #6347: Quick adding of label filter works only once
## Description

Quick adding of label filter works only once

## Reproducible Steps
Go to the reports list.
Click on a label. The associated label filter is added.
Click on an other label. No filter is added.

### LINKED ISSUE #6348: Can't add several values for a filter in Alert Live and Time Line views
Can't add more than one value for each filters in
- Alert Live creation,
- Alert live edition,
- TimeLine views (of Report, Incident, Case RFT, Case RFI)

And
implement new filters UI for Alert Live creation and edition

### PR DESCRIPTION
**Pull request by Archidoit**
PR content:
- issue https://github.com/OpenCTI-Platform/opencti/issues/6348 
- issue https://github.com/OpenCTI-Platform/opencti/issues/6347
- new filters UI in Alert Live creation and Alert live edition

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/analyses/reports/ReportKnowledge.jsx [hunk 1]
```diff
@@ -251,181 +253,189 @@ class ReportKnowledgeComponent extends Component {
...
-              variables={{ id: report.id }}
-              render={({ props }) => {
-                if (props && props.container) {
-                  return <ContainerContent containerData={props.container} />;
-                }
-                return (
-                  <Loader
-                    variant={LoaderVariant.inElement}
-                    withTopMargin={true}
+      <UserContext.Consumer>
```
Comment (labo-flg) [inline opencti-platform/opencti-front/src/private/components/analyses/reports/ReportKnowledge.jsx]: ouch. But probably "a bit" more work if we want to refactor this component and use hooks :)

File: opencti-platform/opencti-front/src/private/components/analyses/reports/ReportKnowledge.jsx [hunk 2]
```diff
@@ -251,181 +253,189 @@ class ReportKnowledgeComponent extends Component {
...
                         this,
                       )}
-                      handleToggleColorsReversed={this.handleToggleColorsReversed.bind(
+                      timeLineFunctionalDate={timeLineFunctionalDate}
+                      handleToggleTimeLineFunctionalDate={this.handleToggleTimeLineFunctionalDate.bind(
                         this,
                       )}
-                      handleToggleModeOnlyActive={this.handleToggleModeOnlyActive.bind(
+                      timeLineFilters={timeLineFilters}
+                      handleAddTimeLineFilter={this.handleAddTimeLineFilter.bind(
```
Comment (SouadHadjiat) [inline opencti-platform/opencti-front/src/private/components/analyses/reports/ReportKnowledge.jsx]: Just from reading the code, it looks like handleAddTimeLineFilter has a different signature now, but we pass it as a parameter to ContentKnowledgeTimeLineBar which then uses it directly to pass it to Filters component (handleAddFilter={handleAddTimeLineFilter}) but handleAddFilter type doesn't have the same signature anymore.

File: opencti-platform/opencti-front/src/private/components/analyses/reports/ReportKnowledge.jsx (hunk 1 above)
Comment (Archidoit) [reply opencti-platform/opencti-front/src/private/components/analyses/reports/ReportKnowledge.jsx]: yes it's far better to use hook... but need refacto ^^'

File: opencti-platform/opencti-front/src/private/components/analyses/reports/ReportKnowledge.jsx (hunk 2 above)
Comment (Archidoit) [reply opencti-platform/opencti-front/src/private/components/analyses/reports/ReportKnowledge.jsx]: fixed :)

**Pull request merged**